### PR TITLE
Add Fall Time key to falling block

### PR DESCRIPTION
**Pull request by clienthax**

### CODE REVIEW

**Comment by JBYoshi** [comment]
Why would you need to use this?

**Comment by clienthax** [comment]
@JBYoshi When spawning a falling block from a block that is air the logic in falling block will instantly destroy the entity,
if you set it to 1 and spawn it in air it will work as intended, this basically makes it possible to spawn the entity anywhere and not just where the original block was located

**Comment by JBYoshi** [comment]
@clienthax It could be fixed by adding code to the implementation of `spawnEntity` to set the fall time to 1.

**Comment by saladoc** [comment]
I think the time a block has been falling is also relevant for calculating damage dealt by and to a falling anvil.

File: src/main/java/org/spongepowered/api/data/manipulator/immutable/entity/ImmutableFallingBlockData.java [hunk 1]
```diff
@@ -76,4 +76,19 @@
      */
     ImmutableValue<Boolean> canDropAsItem();
 
+    /**
+     * Gets the time the block has been falling
+     * if spawning a entity in air this will need to be set to 1 or it will be instantly removed
```
Comment (phroa) [inline src/main/java/org/spongepowered/api/data/manipulator/immutable/entity/ImmutableFallingBlockData.java]: These sentences need periods, and `if` should be capitalized

File: src/main/java/org/spongepowered/api/data/manipulator/immutable/entity/ImmutableFallingBlockData.java [hunk 2]
```diff
@@ -76,4 +76,19 @@
...
+     *
+     * @return The time the block has been falling
+     */
+    ImmutableValue<Integer> fallTime();
+
+    /**
+     * Gets whether this falling block will damage entities where it lands
```
Comment (phroa) [inline src/main/java/org/spongepowered/api/data/manipulator/immutable/entity/ImmutableFallingBlockData.java]: Period

File: src/main/java/org/spongepowered/api/data/manipulator/mutable/entity/FallingBlockData.java [hunk 3]
```diff
@@ -74,4 +74,19 @@
      */
     Value<Boolean> canDropAsItem();
 
+    /**
+     * Gets the time the block has been falling
+     * if spawning a entity in air this will need to be set to 1 or it will be instantly removed
```
Comment (phroa) [inline src/main/java/org/spongepowered/api/data/manipulator/mutable/entity/FallingBlockData.java]: Ditto on this file

**Pull request closed without merging**